src/credentials.rs

```diff
@@ -2,21 +2,20 @@
 
 use eyre::{Context, ContextCompat};
 use serde::{Deserialize, Serialize};
-use std::{env, path::PathBuf};
+use std::{collections::HashMap, env, path::PathBuf};
 use tokio::fs;
 
-use crate::registry::ArtifactoryConfig;
+use crate::manifest::RegistryUri;
```

Comment:
maybe this types is best located in the registry `mod.rs`? it neither strictly belongs in the manifest nor the credentials mod

Reply:
I moved it.